app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def posts
+    @id = params[:user_id]
+  end
```

Comment:
[OPTIONAL]

- I don't know what you wanted to achieve here but this means nothing to me. Please remove that.